### PR TITLE
[Docs] Add alternative install page

### PR DESCRIPTION
**Pull request by evilnick**
When we fix up the navigation (when the other pages are landed) I think we will have individual entries for Windows, macOS etc, but for now we can make do with one link.

### CODE REVIEW

File: docs/install-alternatives.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
+---
+layout: docs
+title: "Alternative Install"
+permalink: /docs/install-alternatives
+---
+
+MicroK8s is spectacularly easy to install and use on Ubuntu or any Linux which
```
Comment (ktsakalozos) [inline docs/install-alternatives.md]: Should there be a header as in all other docs?

File: docs/install-alternatives.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+supports snaps. For other platforms or less common scenarios, see the relevant
+notes below.
+
+## Windows 10
+
+Although Windows 10 now has some very useful features, such as the ability to
+install [Ubuntu as an app][ubuntu-app], the integration even of WSL2 still
```
Comment (ktsakalozos) [inline docs/install-alternatives.md]: is this "even" in the right place?

File: docs/install-alternatives.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+If you wish to experiment with running MicroK8s semi-natively, take a look at
+this [discourse post on WSL2][windows-post].
+
+For now, the best way to run MicroK8s on Windows is with virtualisation.
+MicroK8s will install without problems on Ubuntu running on number of different
+VMs, including [VirtulBox](https://www.virtualbox.org/).
+
+The recommended way to run MicroK8s in a VM on Windows 10 is to use
+[multipass][]. The Windows installer is available for
```
Comment (ktsakalozos) [inline docs/install-alternatives.md]: Should there be a link to multipass here?

File: docs/install-alternatives.md (hunk 2 above)
Comment (evilnick) [reply docs/install-alternatives.md]: yeah, fixed

**Pull request merged**